### ISSUE TITLE
App header icons do not have alt text & not hidden

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4052]
## Area or Component
App Header

## Quick Summary
* The following icons in the app header do not have valid alt text:
    * Person (My Account) icon
    * Settings icon

* The SVG elements for the icons are incorrectly marked as `aria-label=hidden`, making them inaccessible to assistive technology. 
## Accessibility Violation
* `alt-text-missing`

## Action Requested
* Add `role` and `aria-label` attributes to icon HTML elements
* SVG icons should not be hidden
    * Person icon
        * The svg icon for "my account" is marked hidden. The actual link element appears to be focus-able, but it's not given a wrapper or appropriate aria name, so I think the link text would read directly (which is currently "/app/admin#?facility=26f8e3d9-c4b8-49bc-bf30-4c30814ee5dd", which seems both incorrect and useless)
    * Settings icon
        * The svg icon for settings is marked hidden. The actual link is focusable, but like the account button, it doesn't have an accessible name like "settings" and instead just has link text to the admin panel.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/34cd6da49e112854b1c4a0832354e816b228fbd7/frontend/src/app/commonComponents/Header.tsx#L180-L187

https://github.com/CDCgov/prime-simplereport/blob/34cd6da49e112854b1c4a0832354e816b228fbd7/frontend/src/app/commonComponents/Header.tsx#L196

## Original Deque Issue
https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-missing

https://cdc-devtools.dequecloud.com/rule-help/en/semantic-hidden
